apps/balancer-tools/src/app/apr/pool/(components)/HistoricalCharts/HistoricalChart.tsx

```diff
@@ -64,33 +59,6 @@ export default function HistoricalChartWrapper({
     y: [],
   });
 
-  const selectedRoundShape =
```

Comment:
👍 so now we're only showing the data from the selected period. Makes sense to me!